lib/watir-screenshot-stitch.rb

```diff
@@ -14,6 +14,33 @@
 module Watir
   class Screenshot
 
+    #
+    # Employs a cutting edge feature in geckodriver version 0.24.0
+    # to produce a Base64 encoded string of a full page screenshot.
+    #
+    # @warning
+    #   This will fail if geckodriver is > version 0.24.0.
+    #
+    # @info
+    #   This is only a patch until this is baked into Selenium/Watir.
+    #
+    # @example
+    #   browser.screenshot.base64_geckodriver(browser)
```

Comment:
`base64_geckodriver` doesn't require passing `browser` anymore so the yard doc example should be updated here.